### PR TITLE
store: use &str for conf_change_type_str

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing @hhkbp2

### CODE REVIEW

File: src/raftstore/store/util.rs [hunk 1]
```diff
@@ -59,10 +59,13 @@ pub fn check_key_in_region(key: &[u8], region: &metapb::Region) -> Result<()> {
...
 
-pub fn conf_change_type_str(conf_type: &eraftpb::ConfChangeType) -> String {
+const CONF_CHANGE_ADD_NODE_STR: &'static str = "AddNode";
+const CONF_CHANGE_REMOVE_NODE_STR: &'static str = "RemoveNode";
+
+pub fn conf_change_type_str(conf_type: &eraftpb::ConfChangeType) -> &'static str {
     match *conf_type {
-        ConfChangeType::AddNode => "AddNode".to_owned(),
-        ConfChangeType::RemoveNode => "RemoveNode".to_owned(),
+        ConfChangeType::AddNode => CONF_CHANGE_ADD_NODE_STR,
```
Comment (ngaut) [inline src/raftstore/store/util.rs]: s/CONF_CHANGE_ADD_NODE_STR/STR_CONF_CHANGE_ADD_NODE

**Comment by BusyJay** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**